src/coreclr/jit/fgbasic.cpp

```diff
@@ -905,9 +905,16 @@ void Compiler::fgFindJumpTargets(const BYTE* codeAddr, IL_OFFSET codeSize, Fixed
         compInlineResult->NoteInt(InlineObservation::CALLEE_IL_CODE_SIZE, codeSize);
 
         // Determine if call site is within a try.
-        if (isInlining && impInlineInfo->iciBlock->hasTryIndex())
+        if (isInlining)
         {
-            compInlineResult->Note(InlineObservation::CALLSITE_IN_TRY_REGION);
+            if (impInlineInfo->iciBlock->hasTryIndex())
+            {
+                compInlineResult->Note(InlineObservation::CALLSITE_IN_TRY_REGION);
+            }
+            if (impInlineInfo->iciBlock->hasHndIndex())
```

Comment:
What does the `hasHndIndex()` indicate?

Reply:
I read it as "inside EH region (try/catch/finally/filter)"